### PR TITLE
Base build for meteor app

### PR DESCRIPTION
**Pull request by theworkflow**
- Build meteor app using demeteorizer
- Install meteor and demeteorizer as dependencies
- Create function `findFile`
- Remove checking for `npm test` script

### CODE REVIEW

**Comment by jackboberg** [comment]
- [x] test locally
- [x] build RC image
- [x] test on staging

**Comment by theworkflow** [comment]
`get-version` had some errors in it that causes the build to fail. Currently have a PR open for this fix [here](https://github.com/onmodulus/get-version/pull/6).

File: bootstrap.sh [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 
-# overwrite env from base image
+# Overwrite env from base image
 export HOME=/root
 export TEMP_DIR=/tmp
 export TMPDIR=/tmp
 export TMP_DIR=/tmp
 
-#install dependent libraries
+# Install dependent libraries
```
Comment (theworkflow) [inline bootstrap.sh:11]: Do we really need image magick in the build image? It's not being used at all and I don't see it being saved with the bundle.

**Comment by theworkflow** [comment]
Should all the files in the `bin` directory have executable permissions?

**Comment by theworkflow** [comment]
I have this image on the `pc-image-builder` with one manual change to `get-version`. Besides this, the build process completes. Once [this PR](https://github.com/onmodulus/get-version/pull/6) gets merged and an updated version get's published, I can rebuild the image and complete my testing

**Pull request merged**